app/models/job_application.rb

```diff
@@ -0,0 +1,13 @@
+class JobApplication < ApplicationRecord
+  belongs_to :company
+  belongs_to :user
+
+  validates :position_title, presence: true
+  validates :date_applied, presence: true
+  validates :status, presence: true
+  validates :notes, presence: true
```

Comment:
this is one thing that should not be required, in my opinion, as this seems like an optional feature.

Reply:
Been receiving a lot of valid feedback on this, I will change and resubmit.